src/main/java/org/folio/service/RequestExpirationService.java

```diff
@@ -107,8 +107,8 @@ private static Future<List<Request>> getExpiredRequests(AsyncResult<SQLConnectio
       OPEN_AWAITING_PICKUP.value(),
       df.format(new Date()));
 
-
-    String fullTableName = format("%s.%s", PostgresClient.convertToPsqlStandard(tenant), REQUEST_TABLE);
+    String fullTableName = format("%s.%s", PostgresClient.convertToPsqlStandard(
+      pgClient.getTenantId()), REQUEST_TABLE);
```

Comment:
See what `pgClient.getSchemaName()` does, maybe we can use it here instead of `PostgresClient#convertToPsqlStandard`.